app/build.gradle

```diff
@@ -60,9 +60,8 @@ dependencies {
     androidTestCompile 'com.android.support.test.espresso:espresso-core:2.2.1'
     androidTestCompile 'com.android.support.test:runner:0.4.1'
     androidTestCompile 'com.android.support:support-annotations:23.2.1'
-    compile 'com.google.firebase:firebase-auth:9.2.1'
```

Comment:
its already present on lines 48-49 . I dnt know why its not  giving a duplicate error.